packages/pages/src/components/link/link.tsx

```diff
@@ -86,17 +91,28 @@ export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(
     const { children, onClick, className, ...rest } = props;
     const link: CTA = isHREFProps(props) ? { link: props.href } : props.cta;
 
+    let trackEvent: string|null;
+    let analytics: AnalyticsMethods|null;
+
+    if (AnalyticsContext !== null) {
+      trackEvent = props.eventName || props.cta ? "cta" : "link";
+      analytics = useAnalytics();
```

Comment:
hooks can only be called at the top level of the component: https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
probably should add the hooks es lint rule